tests/func-tests/utils.go

```diff
@@ -184,3 +184,15 @@ func UpdateHCO(ctx context.Context, client kubecli.KubevirtClient, input *v1beta
 	}
 	return output, nil
 }
+
+// PatchHCO updates the HCO CR using a DynamicClient, it can return errors on failures
+func PatchHCO(ctx context.Context, cl kubecli.KubevirtClient, patch []byte) error {
+	hcoGVR := schema.GroupVersionResource{Group: v1beta1.SchemeGroupVersion.Group, Version: v1beta1.SchemeGroupVersion.Version, Resource: resource}
+
+	_, err := cl.DynamicClient().Resource(hcoGVR).Namespace(flags.KubeVirtInstallNamespace).Patch(ctx, hcoutil.HyperConvergedName, types.JSONPatchType, patch, metav1.PatchOptions{})
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
WDYT about below:
```suggestion
	return err
```
I understand that some people what is already there, and some prefer what I am suggesting. So this isn't really a request to change, but more of an open question.